tsconfig.json

```diff
@@ -17,7 +17,8 @@
 			"DOM",
 			"ES5",
 			"ES6",
-			"ES7"
+			"ES7",
+			"es2021"
```

Comment:
What's this for?

Reply:
Was getting the error **Property 'replaceAll' does not exist on type 'string'** and adding es2021 fixed it. I will remove